### PR TITLE
Add color palette and images to article and blogs

### PR DESCRIPTION
**Pull request by magmat88**
Responsive display for images after merging to master.
Related issue in progress: #20
Related pull request: #21

### CODE REVIEW

File: src/app/components/aboutPage/aboutPage.component.tsx [hunk 1]
```diff
@@ -1,5 +1,10 @@
 import './aboutPage.component.scss';
 
 export function AboutPage() {
-    return <div>{'about page'}</div>;
+    return (
+        <div>
+            <h2>About page</h2>
+            {'about page content - change me'}
```
Comment (earendillo) [inline src/app/components/aboutPage/aboutPage.component.tsx:7]: No need to put plain string in brackets

File: src/app/components/header/header.component.scss [hunk 2]
```diff
@@ -19,6 +21,20 @@
 
         .navigation-list {
             display: flex;
+
+            a {
+                text-decoration: none;
```
Comment (earendillo) [inline src/app/components/header/header.component.scss:26]: please keep css properties for certain selector in alphabetical order

File: src/app/styles/mixins/mixins.scss [hunk 3]
```diff
@@ -25,12 +24,37 @@
...
+@mixin flexRowCentered {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+@mixin shadowEffect {
+  box-shadow: 5px 5px 10px $color_shadow;
+  border: {
```
Comment (earendillo) [inline src/app/styles/mixins/mixins.scss:45]: change to: border: 1px solid $color_background-light-ivory;

File: src/app/styles/mixins/mixins.scss [hunk 4]
```diff
@@ -25,12 +24,37 @@
...
+    color: $color_background-light-ivory;
+    width: 1px;
+    style: solid;
+  }
+  padding: 10px;
+  margin: 35px;
+  &:hover {
+    padding: 14px;
+    margin: 30px;
+    border: {
```
Comment (earendillo) [inline src/app/styles/mixins/mixins.scss:55]: use inline syntax for border

File: src/app/styles/variables/colors.scss [hunk 5]
```diff
@@ -1,21 +1,15 @@
...
+$color_dark-blue-gray: #5E6882;
+$color_ivory: #FCFCED;
+$color_misty-rose: #FBE2DA;
 
-$color_text-main: #001936;
-$color_text-title: #333;
-$color_text-contrast: #8b8d91;
-$color_text-input: #666;
-$color_text-input-focus: #111;
+$color_background-dark: $color_xiketic;
```
Comment (earendillo) [inline src/app/styles/variables/colors.scss:6]: why you assign variable to variable in CSS? keep it simple please

**Review by earendillo — CHANGES_REQUESTED** [review]

File: src/app/components/aboutPage/aboutPage.component.tsx (hunk 1 above)
Comment (magmat88) [reply src/app/components/aboutPage/aboutPage.component.tsx:7]: brackets removed

File: src/app/styles/mixins/mixins.scss (hunk 4 above)
Comment (magmat88) [reply src/app/styles/mixins/mixins.scss:55]: changed

File: src/app/styles/mixins/mixins.scss (hunk 3 above)
Comment (magmat88) [reply src/app/styles/mixins/mixins.scss:45]: changed

**Pull request merged**